README.md

```diff
@@ -23,20 +23,8 @@ php artisan vendor:publish --provider=RonasIT\\Clerk\\Providers\\ClerkServicePro
 
 3. Add a new `clerk` guard within the `guards` array in your `config/auth.php` file:
 
-```php
-return [
-    'defaults' => [
-        'guard' => 'clerk',
-        'passwords' => 'users',
-    ],
-
-    'guards' => [
-        'clerk' => [
-            'driver' => 'clerk_session',
-            'provider' => 'users',
-        ],
-        // Other guards...
-    ],
+```sh
```

Comment:
let's make config publishing automatically inside the new `install` command and change readme by keeping only required points:
1. Install package
2. Run install command
3. Change config